pkg/controller/volume/expand/expand_controller.go

```diff
@@ -339,7 +333,7 @@ func (expc *expandController) Run(ctx context.Context) {
 	logger.Info("Starting expand controller")
 	defer logger.Info("Shutting down expand controller")
 
-	if !cache.WaitForNamedCacheSync("expand", ctx.Done(), expc.pvcsSynced, expc.pvSynced) {
+	if !cache.WaitForNamedCacheSync("expand", ctx.Done(), expc.pvcsSynced) {
```

Comment:
This should be safe to remove since this controller runs workers over PVCs only, for other PV fetches I saw that it uses the kube client e.g.

```
expc.kubeClient.CoreV1().PersistentVolumes().Get(ctx, volumeName, metav1.GetOptions{})
```